### DISCUSSION TITLE
Implement ModuleFedration

### DISCUSSION THREAD

**Discussion by abhiv01** [iTwin/itwinjs-core#3846] [General]
I need to customise the Webpack to version > 5, as I have to configure Module federation for the twin so that I can integrate it into my application.
Please help me.

**Comment by aruniverse** [comment]
Sorry, I'm not sure what you are asking for, could you please elaborate? 

As of our iTwin.js 3.2.0 release we do have some initial support for Webpack 5 and we're actually using it internally in our test apps in this monorepo. 

You would need to configure module federation and the webpack specific config for that yourself, but it should be doable at the moment if done properly.

**Reply by abhiv01** [reply]
I need to implement Module federation plugin, for that I am creating my own Webpack.config.js and then into that I am doing the changes.
So, if by any other approach I can implement the module federation could you please refer some solutions if possible.

**Reply by aruniverse** [reply]
Your best bet at the moment is to use [@bentley/react-scripts@5.0.0-dev.4](https://www.npmjs.com/package/@bentley/react-scripts/v/5.0.0-dev.4) and then eject from that and then add and configure the [ModuleFederationplugin](https://webpack.js.org/plugins/module-federation-plugin/) yourself. Note the @bentley/react-scripts is still in beta but it works pretty well.
